cmd/utils/flags.go

```diff
@@ -912,12 +912,13 @@ Please note that --` + MetricsHTTPFlag.Name + ` must be set to start the server.
 
 var CrossValidationEndpointFlag = &cli.StringFlag{
 	Name:  "crossvalidation.endpoint",
-	Usage: "http endpoints to cross validate blocks against",
+	Usage: "http endpoint(s) to cross validate blocks against.  Formatted as a comma-separated list of URLs",
```

Comment:
If you turn it into a `StringSliceFlag` you get that functionality for free. It can then be input either as
` --crossvalidation.endpoint="foo,bar,gazonk"` _or_ as `--crossvalidation.endpoint="foo"  --crossvalidation.endpoint="bar"  --crossvalidation.endpoint="gazonk"`